tests/suites/test_suite_ssl.function

```diff
@@ -928,31 +928,86 @@ int mbedtls_move_handshake_to_state( mbedtls_ssl_context *ssl,
 #endif /* MBEDTLS_X509_CRT_PARSE_C */
 
 /*
- * Write application data. Then increase write and fragments counter
+ * Write application data. Increase write counter and fragments counter if
+ * necessary.
  */
 int mbedtls_ssl_write_fragment( mbedtls_ssl_context *ssl, unsigned char *buf,
-                                int ln, int *writen, int *fragments )
+                                int buf_len, int *written,
+                                int *fragments, const int expected_frames )
```

Comment:
I would suggest using the same name for fragments/frames everywhere, hence - renaming `expected_frames` to `expected_fragments`.